docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  offender-assessment-events:
+    build:
+      context: .
+    network_mode: "host"
+    container_name: offender-assessment-events
+    ports:
+      - "8081:8080"
```

Comment:
8080:8080 or you did you not change this from the default?

Reply:
Did'nt change from default

Reply:
As it's a template seems to be easiest to merge as is, then make any changes we want in subsequent PRs. This way the setup is easily documented in git history so we can replicated for the other services